### PR TITLE
Feat/volume meter event

### PR DESCRIPTION
**Pull request by stefandordevic**

### CODE REVIEW

File: conference.js [hunk 1]
```diff
@@ -639,7 +649,10 @@ export default {
         // overshadowed by the overlay.
         tryCreateLocalTracks.then(tracks => {
             APP.store.dispatch(toggleSlowGUMOverlay(false));
-            APP.store.dispatch(mediaPermissionPromptVisibilityChanged(false));
+            if (tracks.length >= 2) {
+                // granted audio and video
+                APP.store.dispatch(mediaPermissionPromptVisibilityChanged(false));
```
Comment (jens-team-ai) [inline conference.js:664]: could this change explain why the prompt didn't disappear in today's meeting (don't remember who ran into this)?

File: conference.js [hunk 2]
```diff
@@ -2663,6 +2714,43 @@ export default {
...
+                            JitsiTrackEvents.TRACK_AUDIO_LEVEL_CHANGED,
+                            subscription
+                        );
+                        jitsiTrack.dispose();
+                        delete _cachedAudioInputTracks[deviceId];
+                    }
+                }
+            });
+        } catch (e) {
+            console.log(e);
```
Comment (jens-team-ai) [inline conference.js]: can you use logger.error instead?

File: modules/API/API.js [hunk 3]
```diff
@@ -1267,6 +1281,43 @@ class API {
...
+     */
+    notifyTrackReceivingStatus(data: Object) {
+        this._sendEvent({
+            name: 'track-receiving-data-status',
+            data
+        });
+    }
+
+    /**
+     * Notify external application (if API is enabled) that a user is talking while he/she is taking.
```
Comment (jens-team-ai) [inline modules/API/API.js]:
```suggestion
     * Notify external application (if API is enabled) that a user is talking while he/she is muted.
```

File: conference.js (hunk 1 above)
Comment (stefandordevic) [reply conference.js:664]: Could be but I believe the issue comes from the app itself where I am conditionally hiding/showing the screen. I will look into it today.

**Comment by stefandordevic** [comment]
@jens-team-ai fixed bug with permissions screen and applied CR advices

**Review by jens-team-ai — APPROVED** [review]

**Pull request merged**